dna/models.py

```diff
@@ -379,7 +378,7 @@ def predict_regression(self, data, *, batch_size, verbose):
         data_loader = self._get_data_loader(data, batch_size, False)
         predictions, targets = self._predict_epoch(data_loader, self._model, verbose=verbose)
 
-        return predictions
+        return predictions, targets
```

Comment:
This is the change that breaks it.  However, the predictions are shuffled so passing them up and evaluating them on the unchanged full dataset will give us different results.